tests/foreman/api/test_capsulecontent.py

```diff
@@ -1708,7 +1708,7 @@ def test_automatic_content_counts_update_toggle(
 
         :BlockedBy: SAT-25503
 
-        :BZ: 2284027
+        :verifies: SAT-26453
```

Comment:
its  a 6.15.z clone of the issue , I'd suggest to keep the original one here 
```suggestion
        :verifies: SAT-25503
```

Reply:
I just searched in the Jira for the BZ and found out this one. Changed.